bouncer/views.py

```diff
@@ -60,6 +60,11 @@ def annotation(self):
         extension_url = "{uri}#annotations:{id}".format(
             uri=document_uri, id=annotation_id)
 
+        parsed_url = parse.urlparse(document_uri)
+        parsed_path = parsed_url.path if parsed_url.path != "/" else ""
+        if parsed_path.endswith("/"): parsed_path = parsed_path[:-1]
```

Comment:
Since we're now stripping the trailing slash on line 65, line 64 can just be `parsed_path = parsed_url.path`